jetson/nav/rover.cpp

```diff
@@ -239,6 +239,7 @@ bool Rover::updateRover( RoverStatus newRoverStatus )
             mRoverStatus.obstacle() = newRoverStatus.obstacle();
             mRoverStatus.odometry() = newRoverStatus.odometry();
             mRoverStatus.target() = newRoverStatus.target();
+            mRoverStatus.target2() = newRoverStatus.target2();
```

Comment:
Why `target` and `target2`? I believe they could be something more descriptive like `leftTarget` and `rightTarget`.

Reply:
Because of how the code works, we don't need to differentiate between left and right targets. When we detect our first target, that is automatically listed as `target`, and if we can detect a second target, that is listed under `target2`. For instance, if we detected the right post first, that would go under `target`, or `leftTarget` with your suggestion (and when we detect the left post, that would go under `rightTarget` because of how our system detects them).

Reply:
When you only see one target, it is the left (and the right) target. My point is that `target` and `target2` have little meaning in terms of adding context. Additionally, the second one spotted won’t always be `target2` just like left wouldn’t always stay as left (or right).

I was suggesting renaming for clarity not because you _needed_ to differentiate it.